pkg/controller/direct/workstations/config_controller.go

```diff
@@ -39,6 +40,40 @@ import (
 
 func init() {
 	registry.RegisterModel(krm.WorkstationConfigGVK, NewWorkstationConfigModel)
+	fuzztesting.RegisterKRMFuzzer(workstationConfigFuzzer())
```

Comment:
Just notice this, could you elaborate how to run this fuzz test if add the init here but not [this file](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/dev/tasks/find-missing-fields)?

Reply:
This seems to be a new(er) method for setting up fuzz tests. I think it's a bit easier to use. The tests are run here https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/95b3225e0040b41db93bb05d26bd4d6007dab44b/.github/workflows/presubmit.yaml#L219, or manually with `go test -v ./pkg/fuzztesting/fuzztests/ -fuzz=FuzzAllMappers -fuzztime 60s`

Reply:
Yeah I believe we have a new framework for fuzzers now. "find-missing-fields" is probably deprecated.

@acpana do we have plan to migrate old-styled fuzz tests and remove the deprecated files to avoid confusion?